epics/stagingTestIOC/configure/RELEASE

```diff
@@ -25,7 +25,7 @@
 # Variables and paths to dependent modules:
 #MODULES = /path/to/modules
 #MYMODULE = $(MODULES)/my-module
-CALC=/epics/support/calc-R3-7-1
+CALC=/epics/support/calc-R3-7-5
```

Comment:
In the docker image which builds EPICS base, there is a line which crates a link at `/epics/base`. This makes me curious whether it would be possible to use the same pattern with other EPICS modules so that we do not need to update the this line when the version of the module changes.